app/src/index.tsx

```diff
@@ -26,6 +26,8 @@ import './index.css';
 import * as serviceWorkerRegistration from './serviceWorkerRegistration';
 import {shouldDisplayRecord} from './users';
 import {localGetDataDb} from './utils/database';
+import {MAP_SOURCE, MAP_SOURCE_KEY, MAP_STYLE} from './buildconfig';
+import {setMapConfig} from '@faims3/forms';
```

Comment:
Would prefer that we configure the components directly with props/config, rather than having a 'register' style workflow. These historically have made the FAIMS code confusing to maintain, as there is no clear link between configuration and the source of that configuration. It also makes testing more confusing as you need to reason about which register function 'runs last'. Also it makes this file 'side effecting' in that it changes behaviour of other modules.